README.md

```diff
@@ -106,6 +106,16 @@ respectively.  This should not be used in production.
 }
 ```
 
+##### disabled     - optional
+
+If `true` `oErrors` will be configure with no operation functions and errors
+will not be reported.
```

Comment:
A few typos.  Also consider making it `enabled` with a default value of true rather than the other way around.